resources/definitions/Ender3_MicroSwiss_DirectDrive.def.json

```diff
@@ -20,8 +20,11 @@
         "extruders_enabled_count": { "value": 1 },
         "machine_use_extruder_offset_to_offset_coords": { "value": true },
         
-        "machine_width": { "value": 230 },
-        "machine_depth": { "value": 230 },
+        "Machine_Dimensions_Comment_1": "Cura Does not calculate Disallowed Areas Properly. Cura shifts zero of build plate to negative-most nozzle.",
+        "Machine_Dimensions_Comment_2": "Workaround is to extend build plate by negative-most offset, then use G92 to put in offsets in start gcode.",
+        "Machine_Dimensions_Comment_3": "Since Nozzle 2 has X offset of -24, Build Width increased by 24mm.",
```

Comment:
This setting doesn't exist. Could you please remove it? Cura gets confused by these things, e.g. when plug-ins list all of the settings.

Comments can currently only be added in Git commit messages, if necessary.